apollo-runtime/src/main/java/com/apollographql/android/ApolloPrefetchCall.java

```diff
@@ -0,0 +1,24 @@
+package com.apollographql.android;
+
+import java.io.IOException;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public interface ApolloPrefetchCall {
```

Comment:
Id like to suggest rename `ApolloPrefetch` it would sound nicer to do client.apolloPrefetch().execute()